docs/build/dapp-staking/indexer.md

```diff
@@ -0,0 +1,112 @@
+# The API and the Indexer
```

Comment:
We should add a disclaimer statement on the line

"..that API is provided as a free endpoint for developers for limited usage and is not suitable for high demand, such as a dApp UI constantly scraping data. As such, anybody should be aware high usage might result in instability of Token API.